lexer/gen/golang/acttab.go

```diff
@@ -65,6 +66,8 @@ type action struct {
 }
 
 const actionTableSrc = `
+// generated by gocc; DO NOT EDIT.
```

Comment:
I think since its an internal API no one should really care if it changes.
But I guess @goccmack should give his opinion.